cf/irsa-worker-role.yaml

```diff
@@ -52,7 +52,11 @@ Resources:
                   - !Sub "arn:aws:s3:::biomage-originals-${Environment}"
                   - !Sub "arn:aws:s3:::worker-results-${Environment}"
                   - !Sub "arn:aws:s3:::processed-matrix-${Environment}"
-                  - !Sub "arn:aws:s3:::biomage-uploaded-samples-${Environment}"
```

Comment:
what is this biomage-uploaded-samples? Are you sure it's not used?

Reply:
yes doesnt exist